### DISCUSSION TITLE
New validations for etsinfv quirk

### DISCUSSION THREAD

**Discussion by jpc4242** [tliron/go-puccini#111] [Ideas]
Hi @tliron : 
So far the etsinfv _meta-quirk_  only activates other _basic_ quirks related to how TOSCA is interpreted.
But ETSI mandates some extra restrictions such as:
 - Some TOSCA.meta fields like `ETSI-Entry-Licenses` are mandatory
 - Only some node type derivations are allowed
 - the vnfd_id (and other fields) appearing both in the Definitions files and in the manifest must match

Do you think it would be interesting to add those validations, or do you prefer puccini to be focused on "plain TOSCA" and defer those validations to external tools?

**Comment by tliron** [comment]
I don't mind at all supporting ETSI variations, especially because they are usually quite simple to implement. Puccini already makes an effort to support all versions of TOSCA since 1.0 as well as various dialects and variations. If ETSI requirements are reasonable we can add them, so feel free to propose some additions.
